website/docusaurus.config.js

```diff
@@ -73,11 +73,9 @@ const config = {
   trailingSlash: isDeployPreview,
   stylesheets: [
     {
-      href: 'https://cdn.jsdelivr.net/npm/katex@0.15.2/dist/katex.min.css',
+      // style lint fails for katex.min.css
```

Comment:
If stylelint fails, add it to the ignore file and commit with `--no-verify`. I have no idea why stylelint isn't respecting the ignore option in priority to the files passed to it during precommit.

Reply:
I see. Not much of a difference using the minified css though.

Reply:
I think we do get a size reduction? Even if it's just a few KBs? Since we never touch the CSS, no need to keep the original :P